packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -51,6 +56,34 @@ export async function activate(context: ExtensionContext): Promise<void> {
     clientOptions
   );
 
+  const jsonSchemaConfig:
```

Comment:
we should comment why we are doing this hack (thiea bug not support https://code.visualstudio.com/api/references/contribution-points#contributes.jsonValidation)